docs/user-guide/docs-source/setup.md

```diff
@@ -0,0 +1,123 @@
+
+![Houdini Launcher License Tab](images/setup/HoudiniInstall_1.1.png)
+
+Once installation completes click on the <number-badge>6</number-badge> "Licenses" tab. Then click the <number-badge>7</number-badge> "License Administrator" button.
+
+![Houdini Launcher License Tab](images/setup/HoudiniInstall_2.png)
+
+Two windows will appear. The "Entitlements" window should display your available "Houdini Engine for Unity/Unreal" licenses. Set the <number-badge>8</number-badge> "Quantity" value to "1" and click the <number-badge>9</number-badge> "Install" button.
+
+![Houdini Administrator Entitlements](images/setup/HoudiniInstall_3.png)
+
+The remaining Houdini Administrator window should now reflect that you've successfully configured your Houdini license. You may close both this window and the Houdini Launcher.
+
+![Houdini Administrator complete](images/setup/HoudiniInstall_4.png)
+
+Next, you'll add the Houdini Engine plugin to your Unreal Engine install. Do this by copying the "HoudiniEngine" folder from...
+
+`C:\Program Files\Side Effects Software\Houdini Engine\Unreal\19.0.x\4.27\HoudiniEngine`
+
+...to the following location...
+
+`C:\Program Files\Epic Games\UE_4.27\Engine\Plugins\Runtime\HoudiniEngine`
+
+
+
+### 4. Add the AWSAmbit.hda to your Houdini Engine install
+
+The final part of setting up Houdini Engine is to copy one of the Ambit plugin files to your Houdini install folder. Recall that you downloaded the Ambit plugin in step 1. Find this file...
+
+`[AwsAmbit plugin folder]\AWSAmbit.hda`
+
+...and copy it to the following location, creating the "otls" directory if necessary...
+
+`C:\Users\[username]\Documents\houdini19.0\otls\AWSAmbit.hda`
+
+
+
+### 5. Install the glTF Exporter UE4 plugin
+
+Some of Ambit's export functionality relies on the **glTF Exporter plugin** from Epic Games. This plugin doesn't ship with Unreal Engine but can be downloaded for free from the Unreal Engine Marketplace following the steps below.
+
+Visit the [glTF Exporter Marketplace web page](https://www.unrealengine.com/marketplace/en-US/product/gltf-exporter).
+
+If not already signed in, click the "Sign in to Buy" button and log in.
+
+Once logged in, click the "Open in Launcher" button. This will take you to the same asset page in the Marketplace tab of the Epic Games Launcher.
+
+Click the "Install to Engine" button and complete the installation.
+
+Congratulations! Initial setup is now complete. 🎉
+
+
+
+## Per Project Setup
+
+Each time you create a new Unreal Engine project in which you'd like to use the Ambit tools you will need to add the Ambit plugin to your project following the steps below.
+
+> ⏱ Project setup takes about 5 minutes.
+
+Navigate to the Unreal Engine project to which you'd like to add Ambit. Inside that project folder create a "Plugins" folder (if it doesn't already exist.)
+
+`[MyUnrealProject]\Plugins\`
+
+Download the latest pre-built release of the Ambit UE4 plugin [here]() 🔥🔥🔥 *TODO: Update link when GitHub is ready.*
```

Comment:
Are you going to manually track this todo? If not, It might be easy for this to slip through the cracks. Might be better to just direct people to the GitHub release page rather than creating a link to the user guide.

Reply:
This was an oversight. The next update to the PR removes the need for this sentence.